packages/api-i18n/src/plugins/service.ts

```diff
@@ -29,6 +29,14 @@ export default () => [
                 return localesCache;
             }
 
+            const urlRegEx = /^(https?:\/\/)((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(\:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(\#[-a-z\d_]*)?$/i;
+
+            if (!urlRegEx.test(process.env.I18N_API_URL)) {
+                throw Error(
+                    `I18N_API_URL env variable is not a valid URL. Make sure it's specified in the service configuration.`
```

Comment:
I would just mention the exact file we are talking about here and that is the `serverless.yml`. This way users will know what exactly we meant, "service configuration" to some users might be confusing.